Makefile

```diff
@@ -241,6 +241,11 @@ doc: install-doc
 	rm ./pachctl
 	mv pachctl.rst doc/pachctl
 
+launch-monitoring:
+	kubectl create -f ./etc/plugin/monitoring
+	sleep 120
```

Comment:
Cool. Yea I almost did just that, but glad it exists somewhere. I like that, I'll add it.